ingestion/src/metadata/ingestion/source/database/snowflake/utils.py

```diff
@@ -121,12 +121,11 @@ def get_view_definition(  # pylint: disable=unused-argument
     schema = schema or self.default_schema_name
     if schema:
         cursor = connection.execute(
-            "SHOW /* sqlalchemy:get_view_definition */ VIEWS "
-            f"LIKE '{view_name}' IN {schema}"
+            f'SELECT /* "app": "OpenMetadata" */ GET_DDL(\'VIEW\',\'{schema}.{view_name}\') AS "text"'
         )
     else:
         cursor = connection.execute(
-            "SHOW /* sqlalchemy:get_view_definition */ VIEWS " f"LIKE '{view_name}'"
+            f'SELECT /* "app": "OpenMetadata" */ GET_DDL(\'VIEW\',\'{view_name}\') AS "text"'
```

Comment:
aren't we including the comments directly on the engine?

Reply:
I think that if we use the connection engine, it'll be handled by this https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/connections/headers.py#L35 so no need to add it explicitely. Note how we don't add it anywhere

Reply:
Ah makes sense thanks for reviewing it. Let me revert it